tdms/tests/unit/test_interp_fns.cpp

```diff
@@ -0,0 +1,292 @@
+# include "catch2/catch_test_macros.hpp"
+# include "interpolation_methods.h"
+
+# include <algorithm>
+# include <cmath>
+# include <iostream>
+
+using namespace std;
+
```

Comment:
loving the tests so far – were you going to add some that test the `interpolateTimeDomain` functions?

Reply:
I have another (local) branch for these tests. I have a couple of tests working already, interpolating some known functions and checking the error in the output matches the error obtained from having MATLAB's `interp` (which BLi is based of). So I'm confident that BLi _is_ doing what it should!

Mostly just waiting on today's (5th) meeting for some ideas surrounding a "BLi beats Cubic interpolation" test, to confirm that BLi is providing an _improvement_ in the parameter regimes we expect.